mirdata/annotations.py

```diff
@@ -79,6 +79,26 @@ def __repr__(self):
         return repr_str
 
 
+class MultiAnnotator(Annotation):
+    """Multiple annotator class.
+    This class should be used for datasets with multiple annotators (e.g. multiple annotators per track).
+
+    Attributes:
+        annotators (list): list with annotator ids
+        annotations (list): list of annotations (e.g. [annotations.BeatData, annotations.ChordData]
```

Comment:
as far as I see in salami, the list of annotations comprises lists of annotations rather than annotations
it's not clear what happens when there are multiple annotation types and multiple annotators 
for instance, is this the structure for two annotators:
[[annotations.BeatData,annotations.BeatData], [annotations.ChordData,annotations.ChordData]]
or this
[[annotations.BeatData,annotations.ChordData], [annotations.BeatData,annotations.ChordData]]

Reply:
in that case, you would have `tracks.chords` and `tracks.beats` and each would return a `MultiAnnotator` class with format `MultiAnnotator(annotations, annotators)`,  e.g. `annotations` being [`BeatData`, `BeatData`] and `annotators` ["annot1", "annot2"]

Reply:
I think the issue is that it's ambiguous from the way the docs are written. 

proposal:
```
list of annotations (e.g. [beat_data1, beat_data2] each with type BeatData or  [chord_data1, chord_data2] each with type chord data
```